go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/unidoc/freetype
+
+go 1.14
+
+require (
+	github.com/golang/freetype v0.0.0-20170609003504-e2365dfdc4a0
```

Comment:
is it not replacing this module?

Reply:
updated 941ed7b3b87bddae8ea85fe3490b06ac7d3bee49